python/packaging/classic/setup.py

```diff
@@ -152,7 +152,7 @@ def _supports_symlinks():
 # python/packaging/connect/setup.py
 _minimum_pandas_version = "1.4.4"
 _minimum_numpy_version = "1.21"
-_minimum_pyarrow_version = "4.0.0"
+_minimum_pyarrow_version = "15.0.2"
```

Comment:
yeah doing it to the latest is too risky. Can you check the release note, and identify which version we should choose?